libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.html

```diff
@@ -29,7 +27,7 @@
     </div>
   </ng-template>
 
-  <!-- loader template with skeleton -->
+  <!-- custom loader with skeleton -->
   <ng-template pTemplate="loader">
     <div *ngIf="showAvatar; else noIconLoader" class="loader-with-icon-container">
```

Comment:
Do "avatar" and "icon" refer to the same concept?

Reply:
yes, I'll change icon to avatar for consistency.